src/contexts/useEmbalmer.ts

```diff
@@ -0,0 +1,49 @@
+// import { ethers } from 'ethers';
```

Comment:
I would name this file `useSarcophagi` since it's retrieving a list of sarcophagi.

Reply:
Yeah. I think when I created it was going to do "embalmer" functions, but this is the only one ... I'll change it.